mappings/net/minecraft/client/texture/NativeImage.mapping

```diff
@@ -27,16 +72,44 @@ CLASS net/minecraft/class_1011 net/minecraft/client/texture/NativeImage
 		ARG 2 width
 		ARG 3 height
 		ARG 5 pointer
```

Comment:
This is a pointer address, not a pointer

Reply:
So... `memoryAddress`? Either way, I don't see why we can't call it `pointer`. They seem like the same thing.

Reply:
We need to agree on a convention for this. Throughout yarn, LWJGL addresses are called `address`, `pointer`, and `id`.

Reply:
Agreed. We also have `handle` that's used by the window, which is technically exactly the same.